sled-agent/src/sim/http_entrypoints_pantry.rs

```diff
@@ -46,11 +52,64 @@ pub fn api() -> CruciblePantryApiDescription {
 // pantry here, to avoid skew. However, this was wholesale copied from the
 // crucible repo!
 
```

Comment:
Is the TODO above here apply to the new code that you added also?  I'm not sure where the "copied from crucible" stuff is here.

Reply:
I copied from `pantry/src/server.rs`, and then changed it a tiny bit to work with the simulated crucible pantry instead.

In the future, I'd like to change the pantry's api to a trait like the other areas of the code, and have both the real and simulated pantry impl that trait.